terraform/modules/iam/README.md

```diff
@@ -0,0 +1,14 @@
+# IAM module for EKS
+
+## Example usage
+
+```terraform
+module "iam" {
+  source         = "../../modules/iam"
+  user_name      = "tf_user"
+  s3_bucket_name = "s3_name"
```

Comment:
everywhere using variables and values set in terraform.tfvars

Reply:
Yeah. But that's just an example.
It is oblivious, that it should be in the variables.